concourse/pipelines/deploy.yml

```
@@ -457,8 +458,9 @@ jobs:
           optional: true
         params:
           AWS_REGION: eu-west-1
-          ASSUME_ROLE_ARN: 'arn:aws:iam::430354129336:role/govuk-concourse-deployer'
+          ASSUME_ROLE_ARN: ((concourse_deployer_role_arn))
           WORKSPACE: ((workspace))
+          GOVUK_ENVIRONMENT: ((govuk_enviroment))
```

There's a typo in the variable name (similarly elsewhere)
```suggestion
          GOVUK_ENVIRONMENT: ((govuk_environment))
```

---

thanks